src/main/scala/org/constellation/consensus/Round.scala

```diff
@@ -71,21 +71,22 @@ class Round(roundData: RoundData, dao: DAO) extends Actor with ActorLogging {
     case msg => log.info(s"Received unknown message: $msg")
   }
 
-  private def scheduleUnionProposals: Cancellable =
```

Comment:
please restrict it to package if you need it at unit test

Reply:
done